lib/msf/ui/console/command_dispatcher/modules.rb

```diff
@@ -971,6 +975,142 @@ def cmd_back(*args)
               driver.destack_dispatcher
             end
           end
+        
+          def cmd_favorite_help
+            print_line 'Usage: favorite [mod1 mod2 ...]'
+            print_line
+            print_line 'Add one or multiple modules to the list of favorite modules stored in the fav_modules file'
+            print_line 'If no module name is specified, the command will add the active module if there is one'
+            print_line
+            print_line 'OPTIONS:'
+            print_line '  -d                Delete the provided module(s) or the active module from the fav_modules file'
+            print_line '  -D                Delete the fav_modules file'
+          end
+
+          #
+          # Helper method for cmd_favorite that writes modules to the fav_modules_file
+          #
+          def cmd_favorite_add(modules, favs_file, contents)
```

Comment:
I know I suggested this name, but while looking back at this, the name of this method follows the naming convention for actual command method names, so perhaps we should change both this one and `cmd_favorite_del()` to `favorite_add()` and `favorite_del()` or something you would prefer.

Reply:
Sure, done!